terraform.tfvars.example

```diff
@@ -104,6 +104,10 @@ operator_state    = "RUNNING"
 operator_timezone = "Australia/Sydney"
 upgrade_operator  = false
 
+### operator volume kms integration
+enable_operator_pv_encryption_in_transit = false
```

Comment:
As above, group related variables together, within the group sort in alphabetical order. If the group is big enough to warrant its own, do a subsection see operator notification below. They all still belong to the operator but the set of parameters below are for operator notification only